web-components/src/components/form/BasketTakeForm.tsx

```diff
@@ -159,9 +154,9 @@ const BasketTakeForm: React.FC<FormProps> = ({
               disabled={!basket.disableAutoRetire}
               className={styles.checkboxLabel}
               label={
-                <Description className={styles.checkboxDescription}>
+                <Subtitle color="primary.contrastText" ml={1}>
```

Comment:
I believe we're missing some `size="lg"` in here
this should look the same as on the CreditSendForm